schema/cf-schema_v1.1.0.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "http://json-schema/org/schema#",
+  "id": "f5schemadb://cf-schema_v1.1.0.json",
+
+  "type": "object",
+
+  "definitions": {
+    "virtualServerType": {
+      "type": "object",
+      "anyOf": [
+        { "required": ["profiles"] },
+        { "required": ["policies"] },
+        { "required": ["sslProfiles"] }
+      ],
+      "properties": {
+        "policies": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/policyType" },
+          "minItems": 1
+        },
+        "profiles": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/profileType" },
+          "minItems": 1
+        },
+        "sslProfiles": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/sslProfileType" },
+          "minItems": 1
+        }
+      },
+      "additionalProperties": false
+    },
+
+    "poolType": {
+      "type": "object",
+      "anyOf": [
+        { "required": ["balance"] },
+        { "required": ["healthMonitors"] }
+      ],
+      "properties": {
+        "balance": {
+          "type": "string",
+          "enum": ["dynamic-ratio-member",
+            "dynamic-ratio-node",
+            "fastest-app-response",
+            "fastest-node",
+            "least-connections-member",
+            "least-connections-node",
+            "least-sessions",
+            "observed-member",
+            "observed-node",
+            "predictive-member",
+            "predictive-node",
+            "ratio-least-connections-member",
+            "ratio-least-connections-node",
+            "ratio-member",
+            "ratio-node",
+            "round-robin",
+            "ratio-session",
+            "weighted-least-connections-member",
+            "weighted-least-connections-node"
+          ]
+        },
+        "healthMonitors": {
+          "type": "array",
+          "minItems": 1,
+          "additionalItems": false,
+          "items": { "$ref": "#/definitions/healthMonitorType" }
+        }
+      },
+      "additionalProperties": false
+    },
+
+    "policyType": {
+      "type": "string",
+      "minLength": 1
+    },
+
+    "profileType": {
+      "type": "string",
+      "minLength": 1
+    },
+
+    "sslProfileType": {
+      "type": "string",
+      "minLength": 1
+    },
+
+    "healthMonitorType": {
+      "type": "object",
+      "oneOf": [{
+        "properties": {
+          "name": { "type": "string", "minLength": 1 }
+        },
+        "required": ["name"],
+        "additionalProperties": false
+      }, {
+        "properties": {
+          "name": { "type": "string", "minLength": 1 },
+          "interval": { "type": "integer", "minimum": 1, "maximum": 86400 },
+          "protocol": { "type": "string", "enum": ["http", "tcp"] },
+          "send": { "type": "string", "minLength": 1 },
+          "timeout": { "type": "integer", "minimum": 1, "maximum": 86400 }
```

Comment:
Where are defaults defined?

Reply:
CCCL holds the defaults based off the protocol.